docs/README.md

```diff
@@ -0,0 +1,44 @@
+# App design docs
+
+This provides an overview of how the calendar works.
+
+
+## Services
+
+![Service diagram](./services.png)
+
+DMS calendar communicates with:
+
+* MariaDB: holds persistent data (e.g. calendar events, attendees, etc.); MySQL-compatible database server
+* DMS Active Directory (LDAP): connects the calendar to DMS accounts used across DMS's other services; account / authentication server
+* Braintree: credit card processesing; third-party service
+* Twilio: sends SMS notifications; third-party service
```

Comment:
Nitpick: maybe swap the Twilio and Sparkpost lines so the order matches the image.

Reply:
Good catch - updated.